nvim/config--nvim--init.vim.symlink

```diff
@@ -3,6 +3,8 @@ set modelines=1
 
 let g:loaded_python_provider = 0
 " let g:loaded_python3_provider = 1
+let g:virtualenv = '~/.virtualenvs/dotfiles-run/bin'
```

Comment:
Check healthcheck

Reply:
Output not clean. But warnings/errors related to testing environment. Not this change.